source/appModules/outlook.py

```diff
@@ -339,12 +341,20 @@ def _generateCategoriesText(appointment):
 		categories = appointment.Categories
 		if not categories:
 			return None
-		# Translators: Part of a message reported when on a calendar appointment with categories
+		bufLength = 4
+		separatorBuf = ctypes.create_unicode_buffer(bufLength)
+		if ctypes.windll.kernel32.GetLocaleInfoW(
+			languageHandler.LOCALE_USER_DEFAULT,
+			languageHandler.LOCALE_SLIST,
+			separatorBuf,
+			bufLength
+		) == 0:
+			raise ctypes.WinError()
+		categoriesCount = len(categories.split(f"{separatorBuf.value} "))

```

Comment:
I hadn't considered this. I had to check the docs to understand this, could you add the following information as a comment:

```
From: https://docs.microsoft.com/en-us/office/vba/api/outlook.appointmentitem.categories
Categories is a delimited string of category names that have been assigned to an Outlook item. This property uses the character specified in the value name, sList, under HKEY_CURRENT_USER\Control Panel\International in the Windows registry, as the delimiter for multiple categories. To convert the string of category names to an array of category names, use the Microsoft Visual Basic function Split.
```

Reply:
Hi @leonardder just waiting on this change. I'll mark this as changes required.

Reply:
```suggestion
		# Categories is a delimited string of category names that have been assigned to an Outlook item.
		# This property uses the user locale's list separator to separate entries.
		# See also https://docs.microsoft.com/en-us/office/vba/api/outlook.appointmentitem.categories
		bufLength = 4
		separatorBuf = ctypes.create_unicode_buffer(bufLength)
		if ctypes.windll.kernel32.GetLocaleInfoW(
			languageHandler.LOCALE_USER_DEFAULT,
			languageHandler.LOCALE_SLIST,
			separatorBuf,
			bufLength
		) == 0:
			raise ctypes.WinError()
		categoriesCount = len(categories.split(f"{separatorBuf.value} "))

```